.github/MAINTAINERS

```diff
@@ -229,6 +229,7 @@ runtime/ftplugin/meson.vim		@Liambeguin
 runtime/ftplugin/modula2.vim		@dkearns
 runtime/ftplugin/modula3.vim		@dkearns
 runtime/ftplugin/mojo.vim		@ribru17
+runtime/ftplugin/mom.vim		@a-vrma
```

Comment:
you are setting someone else here is maintainer? Is @a-vrma okay with this?

Reply:
First, @a-vrma is @avrms that at https://github.com/averms/vim-nroff-ftplugin/pull/1#issuecomment-2564792473 desired such a split off.
Second, it was included in nroff, so I though most prudent to leave the existing nroff maintainers intact and copy headers